eng/deploy.yaml

```diff
@@ -169,25 +169,25 @@ stages:
         ApplicationManifestPath: $(Pipeline.Workspace)/TelemetryApplication/projectartifacts/ApplicationPackageRoot/ApplicationManifest.xml
         ServicesSourceFolder: $(Build.SourcesDirectory)/src/Telemetry/
 
-  - job: deployStatus
-    displayName: Deploy dotnet-status web app
-    dependsOn:
-    steps:
-    - download: current
-      artifact: DotNetStatus
-    - task: AzureRmWebAppDeployment@4
-      inputs:
-        ConnectionType: AzureRM
-        azureSubscription: ${{ parameters.Subscription }}
-        appType: webApp
-        WebAppName: ${{ parameters.DotNetStatusAppName }}
-        deployToSlotOrASE: true
-        ResourceGroupName: monitoring
-        SlotName: staging
-        Package: $(Pipeline.Workspace)/DotNetStatus/DotNetStatus.zip
-        enableCustomDeployment: true
-        DeploymentType: webDeploy
-        RemoveAdditionalFilesFlag: true
+ - job: deployStatus
```

Comment:
as @riarenas notes this change deleted one char of whitespace text and needs to be re-indented